tags/state/marko.json

```diff
@@ -0,0 +1,23 @@
+{
+  "<state>": {
+    "template": "./index.marko",
+    "transformer": "./transformer.js",
+    "@variable": {
+      "type": "expression",
+      "autocomplete": [
+        {
+          "description": "Variable to declare"
+        }
+      ]
+    },
+    "autocomplete": [
+      {
+        "snippet": "state|${1:variable}|",
+        "description": "Use to declare"
```

Comment:
Description is not complete here.

Maybe something like `Used to create an inline controllable state.`